devenv/docker/blocks/prometheus_random_data/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This Dockerfile builds an image for a client_golang example.
 
 # Builder image, where we build the example.
-FROM golang:1.15.1 AS builder
+FROM golang:1.16.0 AS builder
```

Comment:
Is this changed by accident?

Reply:
More like, "changed because i wasn't sure of everything i had to change in order to move to Go 1.16"